op-mode-definitions/show-ssh.xml.in

```diff
@@ -7,6 +7,19 @@
           <help>Show SSH server information</help>
         </properties>
         <children>
+          <node name="dynamic-protection">
+            <properties>
+              <help>Show SSH server dynamic-protection</help>
+            </properties>
+            <children>
+              <node name="attackers">
```

Comment:
As for me, `attackers` is a redundant word
```
show ssh dynamic-protection
```
Could you extend it also for IPv6 `table ip6 sshguard`?

Reply:
`show ssh dynamic-protection` should be sufficient. Please implement the new style op-mode command that supports both raw and human readable output.

See: https://blog.vyos.io/writing-vyos-operational-mode-commands-just-got-better

Reply:
I have made the requested changes.